ema_workbench/em_framework/outcomes.py

```diff
@@ -420,7 +418,9 @@ def __init__(
     def process(self, values):
         values = super().process(values)
         if not isinstance(values, collections.abc.Iterable):
-            raise EMAError(f"outcome {self.name} should be a collection")
+            raise EMAError(
+                f"Outcome {self.name} should be a collection, but is a {type(values)}: {values}"
+            )
         return values
```

Comment:
What's the idea behind `EMAError`? Are there different types of `EMAError`s? Would it be a user mistake, system failure, or can it be both?

Reply:
EMEError is a catchall error for stuff within the workbench going wrong. The main use case is if something goes wrong in the interaction between the workbench and the model we are connecting with. This can be a user mistake but also some system failure.

There is also the CaseError, which is exclusively used if a single experiment fails (_e.g._, a numerical error in a vensim model).

Reply:
Removed printing the full values and changed to `TypeError`.